pkg/integration/integration.go

```diff
@@ -3,12 +3,24 @@ package integration
 import (
 	"fmt"
 	"os"
+	"strings"
 
 	color "github.com/logrusorgru/aurora"
 )
 
 func Print() {
-	fmt.Println(bashSource)
+	var currentShell = os.Getenv("SHELL")
+
+	if strings.HasSuffix(currentShell, "bash") {
+		fmt.Println(shellSource, bashSource)
+	} else if strings.HasSuffix(currentShell, "zsh") {
+		fmt.Println(shellSource, zshSource)
+	} else if {
+		fmt.Fprintln(os.Stderr, color.Red("Your shell configuration is undefined"))
```

Comment:
👍 Maybe mention that the `SHELL` variable itself is empty, and this is what's weird.